aptoide-database/src/main/java/cm/aptoide/pt/database/accessors/InstalledAccessor.java

```diff
@@ -125,12 +126,20 @@ public Completable remove(String packageName, int versionCode) {
         .flatMap(installs -> filterCompleted(installs));
   }
 
+  public Observable<List<Installation>> getInstallationsHistory() {
+    return database.getAll(Installation.class);
+  }
+
   public void insertAll(List<Installed> installedList) {
     database.insertAll(installedList);
+    for (Installed installed : installedList) {
+      database.insert(new Installation(installed));
```

Comment:
This shouldn't be added here. It should be added on the InstallationAccessor (to be created)